src/index.tsx

```diff
@@ -455,6 +456,93 @@ export class App extends React.Component<any, AppState> {
     }
   };
 
+  private getElementContainingPosition = (
```

Comment:
These functions overwrite the external utils that are depending on how the elements actually render on the canvas. Notice we keep all arguments the same except calling `this.getElementsWithZoomScale` on the `elements` argument.